Framework/TestHelpers/inc/MantidFrameworkTestHelpers/MDEventsTestHelper.h

```diff
@@ -343,15 +343,15 @@ static MDGridBox<MDLeanEvent<nd>, nd> *makeMDGridBox(size_t split0 = 10, size_t
   if (nd > 1)
     splitter->setSplitInto(1, split1);
   // Set the size to 10.0 in all directions
-  auto box = std::make_unique<MDBox<MDLeanEvent<nd>, nd>>(splitter);
+  auto box = new MDBox<MDLeanEvent<nd>, nd>(splitter);
```

Comment:
Just want to check, now that this is a raw pointer, does it get manually deleted by the MDGridBox?

Reply:
`MDGridBox` was previously getting passed a raw pointer, so that behaviour is the same, it's just that in the old code the `unique_ptr` would go out of scope when `makeMDGridBox` exited, leaving `MDGridBox` with a dangling pointer.

Reply:
Ah I see. Is it possible to create a unique pointer for `MDBox` and then std::move it into `MDGridBox` so the ownership is transferred to the `MDGridBox`? This way we won't need to worry about deleting the raw pointer when the `MDGridBox` is destructed

Reply:
The BoxController on line 338 is also a raw owning pointer, but as far as I can see it is also never deleted in the destructor for MDBox. Perhaps this might be a maintenance issue to start using unique_ptr's in this part of the code to make sure the lifetime of these objects are managed correctly. I'm happy to approve this if you think it should be solved as part of a separate issue?

Reply:
It looks like in all the tests that use `makeMDGridBox`, at the end of the test the `MDGridBox` and the `BoxController` are deleted manually, e.g. `MDGridBoxTest.h:300`, so yes I think this should be improved a bit, I can make the issue later.

I've changed the `makeMDGridBox` code back to how it was, but disabled the warning just for that method (hopefully I got the correct warning 😆).